packages/http-client-java/generator/http-client-generator-mgmt/src/main/resources/Manager_Configurable.txt

```diff
@@ -108,12 +111,14 @@ public static final class Configurable {
         Objects.requireNonNull(credential, "'credential' cannot be null.");
         Objects.requireNonNull(profile, "'profile' cannot be null.");
 
+        String clientVersion = PROPERTIES.getOrDefault(SDK_VERSION, "{{artifact-version}}");
```

Comment:
If we use `"{{artifact-version}}"`, there may still be diff between code generated with `package-version` and code generated without.

Reply:
Yes. This scenario should be rare, where properties file is missing. I haven't thought of a good way to deal with it.
To pass in `"{{artifact-version}}"` is to at least ensure generated code is correct since we can skip CI codegen check when necessary.

Reply:
Yeah, maybe just do UnknownVersion for now.

We can change in future, if necessary.

Reply:
changed